### PR TITLE
Scheduler: Describe E1058

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

**Review by AntonSermyazhko — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

File: api-reference/10 UI Widgets/Errors and Warnings/E1058.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: ErrorsUIWidgets.E1058
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs in the [Scheduler](/api-reference/10%20UI%20Widgets/dxScheduler '/Documentation/ApiReference/UI_Widgets/dxScheduler/') when the specified [startDayHour](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/startDayHour.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour') is later than the [endDayHour](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/endDayHour.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour').
```
Comment (arminal) [inline api-reference/10 UI Widgets/Errors and Warnings/E1058.md]: the specified xx is later? what do you mean later? what does it do later?

File: api-reference/10 UI Widgets/Errors and Warnings/E1058.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ id: ErrorsUIWidgets.E1058
...
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+To remove this error, specify a **startDayHour** that is earlier than the **endDayHour**,
```
Comment (arminal) [inline api-reference/10 UI Widgets/Errors and Warnings/E1058.md]: same note about ealier

File: api-reference/10 UI Widgets/Errors and Warnings/E1058.md (hunk 1 above)
Comment (RomanTsukanov) [reply api-reference/10 UI Widgets/Errors and Warnings/E1058.md]: These settings specify hours, so if the `startDayHour` is, for example, 8, then the `endDayHour` should be later than 8, for instance, 12.

Comment (arminal) [reply api-reference/10 UI Widgets/Errors and Warnings/E1058.md]: I see. How about this then:
 ...when the specified [startDayHour](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/startDayHour.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour') is set to a time that is later than the [endDayHour]?

File: api-reference/10 UI Widgets/Errors and Warnings/E1058.md (hunk 2 above)
Comment (arminal) [reply api-reference/10 UI Widgets/Errors and Warnings/E1058.md]: You can keep this one the same since the sentence above has more clarity now.

**Pull request merged**